.gitignore

```diff
@@ -1,10 +1,29 @@
-*~ bak .bak  *.bak
-config.cache config.status config.log
+*~
+*.bak
+*/.deps
+*/.libs
 FcgiBin
-Makefile Makefile.in
-aclocal.m4 configure
-fcgi_config.h fcgi_config.h.in
+Makefile*
+aclocal.m4
+autom4te.cache/
+build*
+cgi-fcgi/cgi-fcgi
```

Comment:
same as examples directory, why not add this in .gitignore of cgi-fcgi directory ?